website/static/css/pages/project-page.css

```diff
@@ -1,3 +1,11 @@
+textarea {
+    resize: none;
+}
+
+.citation-name {
+    font-weight: 700;
```

Comment:
You can probably use an OSF helper class for font weight. See "font weight helpers": https://centerforopenscience.github.io/osf-style/style.html#layout-helper